PhysicsTools/PatAlgos/plugins/PATTriggerProducer.h

```diff
@@ -55,7 +55,7 @@
 
 namespace pat {
 
-  class PATTriggerProducer : public edm::EDProducer {
+  class PATTriggerProducer : public edm::stream::EDProducer<> {
```

Comment:
The static analyzer on the IB found this module is connected to the following static

In call stack ' pat::PATTriggerProducer::produce() calls function pat::TriggerObjectStandAlone::packPathNames() static variable pat::TriggerObjectStandAlone::packPathNames()::_warn' is accessed , 'pat::PATTriggerProducer::produce()' overrides 'edm::EDProducer::produce() virtual'

Which is also allowing a call to std::cerr which needs to be fixed: 
http://cmslxr.fnal.gov/source/DataFormats/PatCandidates/src/TriggerObjectStandAlone.cc?v=CMSSW_7_6_X_2015-08-07-2300#0278

Switching the int to std::atomic<int> and changing the std::cerr to edm::LogWarning will solve the problem.